src/main/java/com/demo/MybatisApplication/dto/StudentsDisplayDto.java

```diff
@@ -0,0 +1,13 @@
+package com.demo.MybatisApplication.dto;
+
+import com.demo.MybatisApplication.model.SubjectEntity;
+import lombok.Data;
+
+
+@Data
+public class StudentsDisplayDto {
```

Comment:
```suggestion
public class StudentDisplayDto {
```
Singular!!
Why do we have two dtos with same properties? Can we reuse the dtos?

Reply:
Added to make code more readable. Deleted one of the DTO